src/components/views/settings/tabs/user/GeneralUserSettingsTab.tsx

```diff
@@ -167,7 +171,7 @@ export default class GeneralUserSettingsTab extends React.Component<IProps, ISta
     private async getCapabilities(): Promise<void> {
         const cli = this.context.client!;
 
-        const capabilities = await cli.getCapabilities(); // this is cached
+        const capabilities = cli.getCachedCapabilities() ?? {};
```

Comment:
If I'm understanding this correctly, then if you open user settings before capabilities have been fetched, the controls will stay disabled until you close and re-open settings? `await cli.getCapabilities()` seems preferable in that regard.

Reply:
I agree a spinner is a bit overkill for this corner case, I was just thinking that allowing the controls to unlock eventually is better than leaving them stuck in an inaccurate state (and the method is already async, so, should be low-effort to switch to `getCapabilities`?)

Reply:
As yes true, the method is async anyway - done.